cogs/music.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, bot):
         self.bot = bot
 
         if not hasattr(bot, 'lavalink'):
-            lavalink.Client(bot=self.bot, password=self.bot.lavapass, loop=self.bot.loop, ws_port=1337, shard_count=len(bot.shards))
+            lavalink.Client(bot=self.bot, password=bot.settings.lavapass, loop=self.bot.loop, ws_port=1337, shard_count=len(bot.shards))
```

Comment:
Still needs to have 'self'

Reply:
@devakira False, `bot` is initialized in the snippet of code.

Reply:
Oh well.